prosopopee/prosopopee.py

```diff
@@ -37,13 +39,23 @@ def __init__(self, options):
     def name(self):
         return self.options["name"]
 
-    @property
-    def quality(self):
-        return self.options["quality"]
+    def gm(self, source, target, options):
+        if CACHE.needs_to_be_generated(source, target, options):
+            gm_switches = {
+              "source": source,
+              "target": target,
+              "auto-orient" : "-auto-orient" if options["auto-orient"] else "",
+              "strip": "-strip" if options["strip"] else "",
+              "quality": "-quality %s" % options["quality"] if options.has_key("quality") else "-define jpeg:preserve-settings",
```

Comment:
Since `quality` is in the default options it should always be there no?

Anyway, the "pythonic" way of testing the presence of a key in a dict is

```python
"key" in dict
```

Or, alternatively (this one works for testing on `False`/`None`), depending of what intention you want to transmit:

```python
dict.get("key")
```

Here I think that you should use the second one.